src/searchbar/SearchBar-android.js

```diff
@@ -62,26 +68,33 @@ class SearchBar extends Component {
     const {
       clearIcon,
       containerStyle,
-      leftIcon,
       leftIconContainerStyle,
       rightIconContainerStyle,
       inputContainerStyle,
       inputStyle,
-      noIcon,
+      searchIcon,
+      cancelIcon,
       showLoading,
       loadingProps,
       ...attributes
     } = this.props;
     const { hasFocus, isEmpty } = this.state;
     const { style: loadingStyle, ...otherLoadingProps } = loadingProps;
-    const searchIcon = (
-      <MaterialIcon
-        size={25}
-        color={ANDROID_GRAY}
-        name={hasFocus ? 'arrow-left' : 'magnify'}
-        onPress={hasFocus ? this.cancel : null}
-      />
-    );
+
+    const defaultCancelIcon = {
```

Comment:
defaultProps here too, same for `defaultClearIcon`

Reply:
https://github.com/react-native-training/react-native-elements/pull/1135#discussion_r183971163